### PR TITLE
ref: moving all configs definition into a single file

### PR DESCRIPTION
**Pull request by kbushgit**
Signed-off-by: Kostiantyn Bushko <kostiantyn.bushko@yahoo.com>

### CODE REVIEW

**Comment by patrickvacek** [comment]
Is this just the config changes from https://github.com/advancedtelematic/aktualizr/pull/1688 pulled out into a separate PR? Or which one should we focus on and review/merge first?

File: src/aktualizr_secondary/aktualizr_secondary_config.h [hunk 1]
```diff
@@ -5,12 +5,7 @@
 #include <boost/program_options.hpp>
 #include <boost/property_tree/ini_parser.hpp>
 
-#include "bootloader/bootloader_config.h"
-#include "crypto/keymanager_config.h"
-#include "crypto/p11_config.h"
-#include "logging/logging_config.h"
-#include "package_manager/packagemanagerconfig.h"
-#include "storage/storage_config.h"
+#include "libaktualizr/config.h"
```
Comment (patrickvacek) [inline src/aktualizr_secondary/aktualizr_secondary_config.h:8]: It is slightly unfortunate that we have to include a bunch more stuff in aktualizr-secondary that we won't actually use, but for now that seems like a small issue that we can address at another time.

**Comment by patrickvacek** [comment]
CI only failed because of an issue that Eugene has fixed. Can you rebase?

**Comment by kbushgit** [comment]
> Is this just the config changes from #1688 pulled out into a separate PR? Or which one should we focus on and review/merge first?

this is just refactoring of configs, lets focus on this PR first

**Pull request merged**